core/themes/bartik/template.php

```diff
@@ -24,6 +24,33 @@ function bartik_css_alter(&$css) {
   if (theme_get_setting('color_legacy') && isset($css[$theme_path . '/css/colors.css'])) {
     $css[$theme_path . '/css/colors.css']['data'] = $theme_path . '/css/colors-legacy.css';
   }
+
+  // Remove Bartik's `/css/style.css` if using a custom breakpoint.
+  if (config_get('menu.settings', 'menu_breakpoint') != 'default') {
```

Comment:
This breaks the tests. I locally switched that to:

`if (config_get('menu.settings', 'menu_breakpoint') == 'custom') {` and the menu related tests are passing again.

Reply:
Thanks!